### ISSUE TITLE
False negative: unused variable which has been changed

### ISSUE THREAD

**Issue by jrfnl** [sirbrillig/phpcs-variable-analysis#111]
```php
function foo() {
	$var = 'abc';
	$var = 'def';
	
	$var2  = 'def';
	$var2 .= 'ghi';
	
	$var3  = 10;
	$var3 += 20;
}
```

For the above code, I would expect three errors for unused variables, one each for `$var`, `$var2` and `$var3`, but only `$var` is reported as unused.

Looks like this is related to the use of combined assignment operators.

**Comment by jrfnl** [comment]
FYI: I have a fix ready for this, but need a decision on #108 before I can pull it.
